app/content/blog/wiki/do-not-copy-and-paste-commands-click-the-sticky.md

```
@@ -0,0 +1,23 @@
+In the discord we have bot command channels
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)